src/api/route-services/experiment.js

```diff
@@ -1,16 +1,16 @@
 const _ = require('lodash');
 
+const jsonMerger = require('json-merger');
 const config = require('../../config');
-const mockData = require('./mock-data.json');
-
 const AWS = require('../../utils/requireAWS');
 const logger = require('../../utils/logging');
 const { OK, NotFoundError, BadRequestError } = require('../../utils/responses');
 const safeBatchGetItem = require('../../utils/safeBatchGetItem');
-
 const constants = require('../general-services/pipeline-manage/constants');
 const downloadTypes = require('../../utils/downloadTypes');
 
+const mockData = require('./mock-data.json');
+
```

Comment:
what is this `mockData` doing here?

Reply:
As far as I can tell this was merged in from `master`, maybe check if it's there?

Reply:
Yes, I saw it was there, but it was confusing that we use mock data for a non-test functionality?

Reply:
Not sure why it is. Maybe worth an ask in #engineering?

Reply:
I took a look at the code, this mockData is a very old json, back in the days when we were storing the cell set files in the api and we were using it from inframock. Very old file. It is used only within this `experiment.js` file and I don't think it does anything. Can you just delete it and all its references?

Reply:
removed